### ISSUE TITLE
Subsequent Delta LA values

### ISSUE THREAD

**Issue by martinthomson** [quicwg/base-drafts#307]
The definition of "Delta Last Acked" is the same for all of the timestamp entries.  This implies that every packet number in the timestamp block is expressed relative to the "Last Acked".  That would make it impossible to provide timestamps for packets more than 255 packet numbers behind the last acknowledged packet and multiple ACK frames would be needed to do that.

I don't think that's right, but that subsequent deltas apply to the last timestamp.  That is:
```diff
  packet_number[0] = last_acknowledged - delta_last[0]
- packet_number[n] = last_acknowledged - delta_last[n]
+ packet_number[n] = last_acknowledged - sum(1, n, delta_last[i])
```

**Comment by ianswett** [comment]
Actually, that's how it works today, and I didn't particularly consider it a problem, since TCP and QUIC by default ack every two packets.  One can imagine schemes that ack less often, and I've implemented two of them, but even then, acking over 200 packets at once would be a bit crazy.

There was a choice between ordering the timestamps by time or by packet number.  I ended up choosing time.

Given timestamps are not retransmitted, they're best effort feedback, and the loss of a small number of timestamps cannot be a critical issue.
